code/modules/research/designs.dm

```diff
@@ -604,7 +604,45 @@
 	build_path = /obj/item/weapon/gun/energy/cutter/plasma
 	sort_string = "TAEAC"
 
-
+/datum/design/item/weapon/divet
+	name = "Winchester NK Divet pistol"
+	id = "divet"
+	req_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 5)
+	materials = list(MATERIAL_STEEL = 10000, MATERIAL_SILVER = 4000)
+	build_path = /obj/item/weapon/gun/projectile/divet
+	sort_string = "TACEA"
+
+/datum/design/item/weapon/divetslug
+	name = "divet magazine"
+	id = "divetslug"
+	req_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 4)
+	materials = list(MATERIAL_STEEL = 4000)
+	build_path = /obj/item/ammo_magazine/divet
+	sort_string = "TACEB"
+
+/datum/design/item/weapon/50cal
+	name = ".50 AE magazine"
+	id = "50cal"
+	req_tech = list(TECH_COMBAT = 6, TECH_MATERIAL = 6, TECH_ENGINEERING = 6)
```

Comment:
Requirements too high.

TECH_ENGINEERING can also be removed.

Tech combat 4, and Tech material 5 please.